README.md

```diff
@@ -0,0 +1,65 @@
+Roll 'm Up
+===
+
+This is Flatpak package for Roll 'm Up, classic pinball game. 
+published in Flathub.org: https://flathub.org/apps/details/nl.lostboys.rollemup. 
+
+Roll 'm Up is abandoned proprietary software created by Lostboys Media Lab 
+for Dommelsch Brewery from Netherlands in 1998.
+
+Known issues
+---
+
+- Highscore upload feature does not work: external servers are gone
+  for many years. You can no longer win beer in this game.
+- Resizing window sometimes causes the game to crash. 
+  Please, don't resize it. Content scaling doesn't work anyway.
+- It's impossible to use non-ASCII characters in highscore nickname.
+  Default "Player 1" nickname is fine.
+- Application always crashes when closing. We can live with that.
+
+How to install?
+---
+
+1. Make sure you installed Flatpak and Flathub.org repository: 
+   https://flatpak.org/setup/.
+2. Search for `Roll 'm Up` in GNOME Software or KDE Discover.
+   You can also use this command in terminal:
+   `flatpak install nl.lostboys.rollemup`.
+3. Click `Roll 'm Up` icon in your desktop launcher or run this command:
+   `flatpak run nl.lostboys.rollemup`.
+
+How to build?
+---
+
+Make sure you have Flatpak and `flatpak-builder` installed. 
+
+Install Flathub.org repository user-wide:
+
+```bash
+flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo \
```

Comment:
Similarly, I don't think build instructions are useful here but if it's something you rely on not to forget, it's fine.

Reply:
Yep, that's for me and for people lost in the internet. :)